src/sentry/static/sentry/app/components/sidebar/sidebarItem.jsx

```diff
@@ -69,6 +71,10 @@ class SidebarItem extends React.Component {
 
     const isTop = orientation === 'top';
     const placement = isTop ? 'bottom' : 'right';
+    const TextOverflowHook = HookOrDefault({
```

Comment:
IMO it should be a component outside of the `TextOverflow`, since we don't specifically want to override the overflow wrapper (we want to keep that). Instead I would do something like this:

```js
const LabelHook = HookOrDefault({
    hookName: 'sidebar:item-label',
    defaultComponent: React.Fragment,
});
```

And then 

```jsx
<LabelHook>
    <TextOverflow>{label}</TextOverflow>
</LabelHook>
```

Reply:
That's cleaner- updated!